frontend/src/components/sustainability/SustainabilityTable.js

```diff
@@ -1,49 +1,124 @@
 import React from 'react';
 import '../Table.css';
-import {MAX_NON_BILLABLE_TO_BILLABLE_RATIO} from "../../constants";
+import {MAX_NON_BILLABLE_TO_BILLABLE_CELL_RATIO, MAX_NON_BILLABLE_TO_BILLABLE_RATIO} from "../../constants";
 
 const statusClass = (remaining) => Math.round(remaining) >= 0 ? 'on-track' : 'overtime';
 
-const SustainabilityTable = ({accounts}) =>
+const overall_headers =
+    <>
+        <td>
+            Billable hours
+        </td>
+        <td>
+            Total non-billable hours
+        </td>
+        <td>
+            % non-billable
+        </td>
+    </>;
+
+const overall_fields = accounts =>
+    <>
+        <td>
+            {Math.round(accounts.billable)}
+        </td>
+        <td>
+            {Math.round(accounts.non_billable_total)}
+        </td>
+        <td className={statusClass(accounts.remaining)}>
+            {Math.round(accounts.total_ratio)}% (max {MAX_NON_BILLABLE_TO_BILLABLE_RATIO * 100}%)
+        </td>
+    </>;
+
+const overall_hints =
+    <>
+        <li>Total non-billable hours = non-billable cell hours + non-billable non-cell hours</li>
+        <li>% non-billable = total non-billable hours / total hours</li>
+    </>;
+
+const cell_headers =
+    <>
+        <td>
+            Non-cell hours
+        </td>
+        <td>
+            Billable cell hours
+        </td>
+        <td>
+            Non-billable cell hours
+        </td>
+        <td>
+            % non-billable cell
+        </td>
+        <td>
+            Remaining non-billable hours
+        </td>
+    </>;
+
+const cell_fields = accounts =>
+    <>
+        <td>
+            {Math.round(accounts.non_billable)}
+        </td>
+        <td>
+            {Math.round(accounts.billable)}
+        </td>
+        <td>
+            {Math.round(accounts.non_billable_responsible)}
+        </td>
+        <td className={statusClass(accounts.remaining_responsible)}>
+            {Math.round(accounts.responsible_ratio)}% (max {MAX_NON_BILLABLE_TO_BILLABLE_CELL_RATIO * 100}%)
+        </td>
+        <td className={statusClass(accounts.remaining_responsible)}>
+            {Math.round(accounts.remaining_responsible)}
+        </td>
+    </>;
+
+const cell_hints = accounts =>
+    <>
+        <li>Total hours = non-cell hours + cell hours</li>
+        <li>Non-cell hours = hours logged on a task not belonging to the current cell</li>
```

Comment:
As mentioned in the forum, this description can be improved to clarify whether `non-cell` means „accounts whose management wasn't delegated“ or it means „logged by people from the other cell“. Either description and name can be improved. If you know a good one you can do it without waiting for the forum results. Otherwise we can leave it as it is (though unclear) and improve it later when we discuss it.

Reply:
I find "accounts whose management wasn't delegated" confusing from the point of view of a person without specific context (e.g. looking at this board for the first time). I think we should come up with a little more user-friendly name for this. so I'll leave it now.